server/src/main/java/org/opensearch/cluster/routing/allocation/AwarenessReplicaBalance.java

```diff
@@ -31,19 +33,55 @@
  * Helps in balancing shards across all awareness attributes and ensuring high availability of data.
  */
 public class AwarenessReplicaBalance {
+    public static final String SETTING_CLUSTER_ROUTING_ALLOCATION_AWARENESS_BALANCE = "cluster.routing.allocation.awareness.balance";
+    public static final String SETTING_USE_FORCE_ZONE_FOR_REPLICA = "cluster.use_force_zone_for_replica";
```

Comment:
rename to `cluster.routing.allocation.awareness.force_replica` ? 

@Bukhtawar  : your thoughts on this name ?

Reply:
maybe `force_zone_replica`? Just `force_replica` might be confusing.

Reply:
How about `cluster.routing.allocation.enforce.awareness_replicas`